cmd/kube-scheduler/app/server_test.go

```diff
@@ -396,7 +396,6 @@ leaderElection:
 			flags: []string{
 				"--leader-elect=false",
 				"--leader-elect-lease-duration=2h", // CLI args are favored over the fields in ComponentConfig
-				"--lock-object-namespace=default",  // deprecated CLI arg will be ignored if --config is specified
```

Comment:
there is no deprecated flag scenario covered in this testcase now, how about set another deprecated flag here? e.g. "--profiling=true"

Reply:
Fixed